proteuslib/unit_models/nanofiltration_0D.py

```diff
@@ -416,26 +948,32 @@ def initialize(
             optarg : solver options dictionary object (default=None)
             solver : str indicating which solver to use during
                      initialization (default = None)
+            fail_on_warning : boolean argument to fail or only produce  warning upon unsuccessful solve (default=False)
+            ignore_dof : boolean argument to ignore when DOF != 0 (default=False)
 
         Returns: None
         """
         init_log = idaeslog.getInitLogger(blk.name, outlvl, tag="unit")
         solve_log = idaeslog.getSolveLogger(blk.name, outlvl, tag="unit")
         # Set solver options
+        if optarg is None:
+            optarg = {'nlp_scaling_method': 'user-scaling'}
```

Comment:
In general, we should move away from 'user-scaling' for the solver. However, if this is the only way to get this to solve for now, then I guess we need to keep it.

Reply:
Certainly- thank you for discovering the discrepancy yesterday as this was something we were overlooking. @bknueven made some nice progress with adjusting scaling factors by probing extreme Jacobian entries, but this will likely be reserved for a subsequent PR , once we confirm the logic/stability of newly adopted scaling factors.